src/Swagger/Extractor/CollectionGetExtractor.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ApiPlatform\Core\Swagger\Extractor;
+
+use ApiPlatform\Core\Metadata\Resource\Factory\ResourceMetadataFactoryInterface;
+use ApiPlatform\Core\Metadata\Resource\ResourceMetadata;
+use ApiPlatform\Core\Swagger\Model\SwaggerDocumentation;
+use ApiPlatform\Core\Swagger\Model\SwaggerMetadata;
+use ApiPlatform\Core\Swagger\Util\SwaggerDefinitions;
+use ApiPlatform\Core\Swagger\Util\SwaggerFilterDefinitions;
+use Symfony\Component\HttpFoundation\Request;
+
+class CollectionGetExtractor implements SwaggerExtractorInterface
+{
+    /**
+     * @var SwaggerExtractorInterface
```

Comment:
IDE and static analysis tools infer the type from the constructor type hint. The Symfony CS (that we follow) tells to not add them (it eases the maintenance).